### ISSUE TITLE
Scrollbar in certain dropdown menus loses focus when clicked

### ISSUE THREAD

**Issue by mabryl** [ckeditor/ckeditor5#12902]
## 📝 Provide detailed reproduction steps (if any)

1.  Go to [this section](https://ckeditor.com/docs/ckeditor5/latest/features/special-characters.html#adding-a-new-category) of our docs and scroll to the interactive demo
2.  Click the Special characters button and then on the All button
3.  Try clicking on the scrollbar that appears next to the category names

### ✔️ Expected result

You should be able to click and hold the left mouse button to drag the list. The gif below is from v35.1.0 where everything works as expected:

![35 1 0](https://user-images.githubusercontent.com/72079603/203316688-8a988704-b65a-4aa6-a4cf-71f084c5f9e5.gif)

### ❌ Actual result

You're not able to click the scrollbar as it immediately loses focus. Scrolling with the mouse wheel works fine. Bug is present in v35.2.0 and up:

![35 2 0](https://user-images.githubusercontent.com/72079603/203317025-21dcbe0b-ebd9-4a2a-8381-7af686b66d0d.gif)

### ❓ Possible solution

I noticed that v35.1.0 didn't have `tabindex = "-1"` set on the div in the screenshot below but later versions did. Removing it from the markup fixed the issue in the latest version of our docs.

![image](https://user-images.githubusercontent.com/72079603/203318252-acdad883-d35e-4064-ad83-8605bf5d02f0.png)

## 📃 Other details

*   Browser: cross-browser
*   OS: macOS
*   First affected CKEditor version: 35.2.0
*   Installed CKEditor plugins: N/A

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mlewand** [comment]
I didn't pin down precise version when it was fixed, but it's not reproducible in 39.0.1 🎉

I was able to reproduce it in 38.1.0 (I didn't check if there were any release in-between).
